tests/wpt/mozilla/tests/css/transform_skew_ref.html

```diff
@@ -17,15 +17,15 @@
 }
 
 .transformed1_ref {
-    transform: matrix(1, 0, 0.30933624961, 1, 0, 0);
+    transform: matrix(1, 0, 0.30933626267, 1, 0, 0);
 }
 
 .transformed2_ref {
     transform: matrix(1, 0.54630248984, 0, 1, 0, 0);
 }
 
 .transformed3_ref {
-    transform: matrix(1, 0.54630248984, 0.30933624961, 1, 0, 0);
+    transform: matrix(1, 0.54630248984, 0.30933626267, 1, 0, 0);
```

Comment:
I will double check this part tomorrow. The value should be the tan() value of 0.3.
```0.3f64.tan() == 0.30933624961```
```0.3f32.tan() == 0.30933627```
```(0.3f32 as f64).tan() == 0.30933626267``` <- This is what I got.

I think the the casting makes we lose something, i.e. ```(0.3f32 as f64) == 0.30000001192092895507812500000000```, instead of ```0.3```.

Reply:
An interesting thing is, I also got an unexpected pass after updating this ref file on Linux. So looks like this problem has been in existence for a long time.
```
[transform_skew_a.html]
  type: reftest
  expected:
if os == "linux": FAIL
```

Reply:
This float point issue could be avoid by using ```1/pow(2, n)```, i.e. 0.5, 0.25, so I will replace ```skewX(0.3rad)``` with ```skewX(0.25rad)``` to avoid this annoying issue.